Plugins/IMA/IMAPlugin.swift

```diff
@@ -366,15 +366,14 @@ extension IMAAdsManager {
             }
         case .STARTED:
             if let ad = event.ad {
-                let adInfo = AdInfo(ad: ad)
+                let adInfo = PKAdInfo(ad: ad)
```

Comment:
@gal-orlanczyk  let's make sure Android team have same prefix for PKAdInfo class.

cc @OrenMe